### ISSUE TITLE
Accessibility issue - cannot drag the widgets by use of keyboard arrow keys 

### ISSUE THREAD

**Issue by johnli388** [tiberiuzuld/angular-gridster2#886]
Similar to this example: https://salesforce-ux.github.io/dnd-a11y-patterns/#/?_k=z48zja

Gridster2 needs to support keyboard arrow keys to drag the widgets.
